tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -339,4 +348,57 @@ public void UnsubscribeFromWidgetUpdates()
     {
         Widget.WidgetUpdated -= HandleWidgetUpdated;
     }
+
+    private void AnnounceWarnings(AdaptiveCard card)
+    {
+        foreach (var element in card.Body)
+        {
+            SearchForWarning(element, false);
+        }
+    }
+
+    private void SearchForWarning(IAdaptiveCardElement element, bool isInsideWarningContainer)

```

Comment:
This method recursively searches through every element in a widget to find every container with a "warning" style.  Does that sound right?

Reply:
Yes, this assumes that any text inside a warning style container should be announced on a widget.